src/ui/Omnibox/Omnibox.ts

```diff
@@ -239,6 +240,15 @@ export class Omnibox extends Component {
     numberOfSuggestions: ComponentOptions.buildNumberOption({
       defaultValue: 5,
       min: 1
+    }),
+    /**
+     * Specifies the number of characters that should be written before displaying the suggestions
+     *
+     * Default value is `0`. Minimum value is `0`.
+     */
+    characterThresholdForSuggestions: ComponentOptions.buildNumberOption({
+      defaultValue: 0,
```

Comment:
We should probably add:

`depend: 'enableQuerySuggestAddon'`

(as I understand it, this only applies to Coveo ML query suggestions, not field-based suggestions, right?)

Reply:
Indeed.

Reply:
I will add it